src/components/src/ProductListItem/productlistitem.main.tsx

```diff
@@ -148,7 +148,15 @@ class ProductListItemMain extends Component<ProductListItemMainProps, ProductLis
         <div className="category-item-inner">
           <div className={`category-item-thumbnail-container ${imageStatus === 'loaded' ? 'loaded' : ''}`}>
             <Link to={`${itemDetailLink}/${encodeURIComponent(productData._code[0].code)}`}>
-              <ImageContainer className="category-item-thumbnail img-responsive" isSkuImage fileName={productData._code[0].code} imgUrl={Config.skuImagesUrl.replace('%sku%', productData._code[0].code)} onLoadData={() => { this.handleImageLoaded(); }} />
+              <ImageContainer
+                alt={`category item thumbnail image for ${productData._code[0].code}`}
+                imgClassName="category-item-thumbnail img-responsive"
+                isSkuImage
+                fileName={productData._code[0].code}
+                imageFileTypes={['webp', 'jp2', 'jpg']}
```

Comment:
Can we take the "imageFileTypes" from ep.config.json? We're duplicating this across components but it's a shared config and could be changed by an implementor.

Reply:
I would +1 this!

Reply:
^ Overriding it per usage sounds like a good idea to me!  I'll make those changes :)

Reply:
Thanks!

Reply:
updated